editor/edit-post/modes/keyboard-shortcuts/map.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
Thanks for the changes 👍 

If this map is supposed to grow over time and contain all edit-post shortcuts, should we move it to `editor/edito-post/keyboard-shortcuts.js` instead?

Reply:
Yea, probably :) I was confused, since I already have `modes/keyboard-shortcuts` folder and didn't want to create clutter. I will move them there.